lib/cocoapods-downloader/git.rb

```diff
@@ -105,7 +105,7 @@ def clone_arguments(force_head, shallow_clone)
       #
       def checkout_commit
         Dir.chdir(target_path) do
-          git! 'checkout', '-b', 'activated-commit', options[:commit]
+          git! 'checkout', '--quiet', options[:commit]
```

Comment:
meh I really don't think there's any value in `checkout`s output, and it's still informative when it fails.